app/controllers/navigation/Navigator.scala

```diff
@@ -393,9 +394,12 @@ object Navigator {
 
   private def documentsProducedPreviousPage(cacheModel: ExportsDeclaration, mode: Mode, itemId: String): Call =
```

Comment:
Since you've changed 'DocumentsProduced' to 'AdditionalDocuments', please go through the whole service and change all places where this term was used.